x-pack/plugins/security_solution/public/cases/attachments/external_reference_event.test.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
How come you chose `enzyme`?
(optional) We all kind of just settled on using react testing library, so maybe consider that instead of using `enzyme`

Reply:
I saw enzyme being used in many places in Kibana. If you prefer I can refactor that to use RTL :)

Reply:
I personally prefer react testing library just because it seems to be used much more in our code base. You can continue to use enzyme if you like.